server/src/instant/model/app_member_invites.clj

```diff
@@ -80,15 +80,16 @@
 
 (defn reject-by-email-and-role
   ([params] (reject-by-email-and-role aurora/conn-pool params))
-  ([conn {:keys [inviter-id invitee-email role]}]
+  ([conn {:keys [inviter-id app-id invitee-email role]}]
```

Comment:
Realized we needed the app-id here, otherwise we would reject all invites to this user by this inviter, rather than just for the app